meta-lmp-base/recipes-containers/composeapp/composectl_git.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "A CLI utility to manage compose apps"
+
+SECTION = "devel"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=2a944942e1496af1886903d274dedb13"
+
+GO_IMPORT = "github.com/foundriesio/composeapp"
+GO_IMPORT_PROTO ?= "https"
+SRC_URI = "git://${GO_IMPORT};protocol=${GO_IMPORT_PROTO};branch=main"
+SRCREV = "38874e6afbf2a2f86171bae2509096abacadfd83"
+UPSTREAM_CHECK_COMMITS = "1"
+
+inherit go-mod
+
+GO_INSTALL = "${GO_IMPORT}/cmd/composectl"
+GO_EXTRA_LDFLAGS = "\
+    -X '${GO_IMPORT}/cmd/composectl/cmd.storeRoot=/var/sota/reset-apps' \
+    -X '${GO_IMPORT}/cmd/composectl/cmd.composeRoot=/var/sota/compose-apps' \
+    -X '${GO_IMPORT}/cmd/composectl/cmd.overrideConfigDir=/usr/lib/docker' \
+"
+
+#do_install:append() {
+# TODO
+#}
```

Comment:
Mind moving the change from the commit that follows to fix the do_install from this commit?

Please also update the commit body message explaining the commit.

Reply:
I think I better just remove these three commented lines, since this commit is intended solely for adding of `composectl` installation. The next commit does the switch and requires adding the symlink.

Reply:
updated.